missioncontrol/v0/time.py

```diff
@@ -28,6 +30,19 @@ def add_seconds(t, s):
         return timescale.utc(*map(sum, zip(t.utc, (0, 0, 0, 0, 0, s))))
 
     def utc(t):
+        """ do whatever it takes to make time into datetime
+        """
+        if t == "now":
+            t = datetime.utcnow()
+        elif isinstance(t, datetime):
+            pass
+        elif isinstance(t, str):
+            t = parse(t)
+        elif isinstance(t, tuple):
+            t = datetime(*t)
+        return t.replace(tzinfo=UTC)
```

Comment:
Not super happy on this - what if the time passed in has a timezone on it - maybe check if it's naive and if so set it to UTC.

Reply:
Alternatively - `t.astimezone(UTC)` ?

Reply:
I've updated this function.
It will now assume naive datetimes are UTC, and raise a ValueError for any non-UTC tz-aware datetimes.

Reply:
Importantly, we **do not** accept non-UTC iso8601 times. We want to enforce iso8601t/z/dot as an interchange format.